internal/gatewayapi/translator.go

```diff
@@ -1008,6 +1012,157 @@ func (t *Translator) ProcessHTTPRoutes(httpRoutes []*v1beta1.HTTPRoute, gateways
 								"RequestHeaderModifier Filter did not provide valid configuration to add/set/remove any headers",
 							)
 						}
+					case v1beta1.HTTPRouteFilterResponseHeaderModifier:
+						// Make sure the header modifier config actually exists
```

Comment:
can this code be moved into a function so it can be reused by request headers as well ?

Reply:
I suggest keeping this for now, and another pr to refactor the processing http routes. Not only the response header modifier, but also other filters could be moved to a func.

Reply:
sounds good, can you please raise a issue to track this